pages/common/wget.md

```diff
@@ -4,11 +4,11 @@
 > Supports HTTP, HTTPS, and FTP.
 > More information: <https://www.gnu.org/software/wget>.
 
-- Download the contents of an URL to a file (named "foo" in this case):
+- Download the contents of a URL to a file (named "foo" in this case):
```

Comment:
Will you please undo this and the next change of `an` → `a`?  It's definitely correct, but it's already covered by @SethFalco in #6069 [here](https://github.com/tldr-pages/tldr/pull/6069/files#diff-c74d3e683274beb25808615449f2ae71adb378f031806a8e85a96d9f26f90b7f), and will cause conflicts if it's fixe here as well.
```suggestion
- Download the contents of an URL to a file (named "foo" in this case):
```